PrioritiesSprint3.md

```diff
@@ -0,0 +1,67 @@
+
+Each priority has a list of subtasks that are to be completed within this sprint.  
+
+Note that ARAs will be required to implement either MCQ or Pathfinder (or both).
+
+# Develop Pathfinder Query
+## Pre-development work:
+- Design new landing page to distinguish types of interactions (templated query, pathfinder, multiquery(?)) (UI)
+- Design query selection interface - Dependencies: decisions on categories of nodes to be allowed as starting points and any other factors allowed for query selection (UI, ARAs)
+- User testing path grouping strategies (UI, ARAs)
+- Design results page path  and graph display (similar to results now) - Dependencies: understanding of the number of paths and how they will be grouped (UI, ARAs)
+- Research to identify new facets needed. e.g. facets for other categories - Dependencies: pathfinder data examples (UI, ARAs)
+- User research filter designs for new filters like metaedge type or other. (UI)
+- Provides example datasets to UI (ARAs)
+## Octopus (CI) work:
+- Evaluate performance and result quality from prototype implementations (ARAs)
+
+# MultiCURIE Query (MCQ)
+## Pre-development work:
+- Designs multi curie selection -  requires interface to add and subtract select curies (how many max?) (UI)
+- Investigate additional facets needed ( e.g. curie match facet, other specific to starting or ending node category if neccesary) (UI)
+- User research filter designs for new filters like metaedge type or other. (UI)
+- ## Eel (CI):
+- Relevant KPs implement lookup MCQ (KPs)
+- NodeNorm implements set ID method
+## Octopus (TEST):
+- TRAPI 1.5 finalized and released (ARAs, KPs, UI)
+
+# Display Full KL/AT in the UI
+## Pre-Development
+- ARAs/O&O plans for utilizing KL/AT information in their scoring approaches (ARA, O&O)
+- Define next EPC area to standardize and implement in data and UI. (EPC)
+- ## Eel (CI):
+- Last few KPs/ARAs annotate edges with KL/AT terms (See status table [here](https://docs.google.com/document/d/16agzJB0OlR8z-zU3nTivYYIo09sdayN-tu8lbrZyTXY/edit?pli=1#bookmark=id.kz0zera1i398)) (KPs, ARAs)
+## Octopus (TEST):
+- UI team implements full KL/AT tags in UI display (define paradigm to accommodate this info, which may include user friendly synonyms, new icons/badges, updated EPC modal tables, etc). (UI)
+- Determine how to display KL/AT for KPs/ARAs that have not yet implemented edge-level annotations (may fall back to infores-based annotation?) (UI)
+
+# Treats Refactor
+## Octopus (TEST):
+- UI implements "treats" refactor including provenance (UI)
+
+# Scalability
+## Eel (CI):
+- Sending Concurrent queries (MVP1&MVP2) in batch of 5,10,15,20,25,30 (Test Env)
+
+# CQS Workflows
+## Eel (CI):
+-  Test and iterate MVP1 workflows
+-  Push these workflows to TEST
+
+# Fix Failing Automated Tests
+The current number of tests is 484, so the number of new tests is (Number of tests - 484)
```

Comment:
I'm not entirely sure who added the numbers here. But I do think we need to iron out both what the realistic goals should be, who is actually in charge of meeting them, and is there any consequence for not meeting the goals? (i.e. if an ARA has too many failing tests, what happens? Are they excluded from the ARS until they are in compliance?) Right now, we have some goals that no ARA is currently meeting, and from the Testing team perspective, I'm not really sure what I can do about that other than provide the report.

Reply:
Yeah, I'm really trying to unmix the ARA target from the changes that the testing group made, which makes it hard to provide concrete numbers

Reply:
Sorry if I'm using the wrong testing terminology, but is there a way to create a test suite of the current test assets and define our performance goals relative to just that test suite? Hopefully that way, when new tests are added, it doesn't change the goal posts.

Reply:
@maximusunc what do you think?

Reply:
That's an excellent idea and very easy to implement. Let's go that route and have some concrete numbers for this sprint goal.

Reply:
So it looks like there are as of right now 604 tests. We can pin that and use it as the Sprint 3 Test Suite. So maybe we make the goals (I think I still like the numbers I posted above?):
- ARAs must pass at least 350 tests
- ARAs must provide results for at least 425 tests
- The CQS must pass at least 100 tests

Reply:
Great, sounds good. I'll update the PR

Reply:
OK, PR updated

Reply:
> 
> The overall idea would be that ARAs and CQS would meet the targets which we would then progressively make more difficult over time. So I think the Testing team is meeting what it needs to do, the impetus is on the ARAs at this point.
> 


Chris, this is important but I am not sure what we mean by "more difficult" - if it is based on the # of tests passed, then it is self-referential. My concern is: How will the current performance generalize beyond these MVP1/2 cases?  The test assets are very biased and do not cover a large variety of queries - they are quite dependent (derived) from each other .... Maybe for discussion of next testing in next sprint.